aliveim.go

```diff
@@ -56,7 +62,11 @@ func handleAlivePost(rw http.ResponseWriter, request *http.Request) {
 	} else {
 		timer := time.NewTimer(time.Millisecond * time.Duration(aliverequest.Timeout))
 		deviceTimer := DeviceTimer{aliverequest.DeviceID, timer}
+
+		mutex.Lock()
 		timersMap[aliverequest.DeviceID] = deviceTimer
+		mutex.Unlock()
+
 		go deviceTimer.startTimer()
```

Comment:
Note: At this point there could be 2 requests resetting the same timer. Is that acceptable? Needs some thinking